package.json

```diff
@@ -7,17 +7,24 @@
     "npm": "10.8.2"
   },
   "scripts": {
+    "predev": "test ! -d getgather/frontend/__assets && npm run build || echo 'Assets directory exists, skipping build'",
     "dev": "concurrently --raw \"npm run dev:backend\" \"npm run dev:frontend\"",
     "dev:backend": "uv run -m uvicorn getgather.main:app --reload --host 127.0.0.1 --port 23456",
     "inspector": "ALLOWED_ORIGINS=http://localhost:5173,http://localhost:6274 npx @modelcontextprotocol/inspector",
     "dev:frontend": "concurrently --raw \"npm run watch:schema:api\" \"vite\"",
     "build": "tsc -b && vite build",
-    "lint": "eslint .",
     "preview": "vite preview",
-    "check-format": "prettier --check \"**/*.{html,js,jsx,ts,tsx,css,json,md}\"",
-    "format": "prettier --write \"**/*.{html,js,jsx,ts,tsx,css,json,md}\"",
-    "schema:api": "wait-on tcp:23456 && openapi-typescript http://localhost:23456/api/openapi.json -o frontend/__generated__/api.d.ts",
-    "watch:schema:api": "nodemon --watch getgather -e py --exec \"npm run schema:api\""
+    "frontend:check:format": "prettier --check \"**/*.{html,js,jsx,ts,tsx,css,json,md}\"",
+    "frontend:format": "prettier --write \"**/*.{html,js,jsx,ts,tsx,css,json,md}\"",
+    "frontend:lint": "eslint .",
+    "backend:check:format": "uv run ruff check && uv run ruff format --check",
+    "backend:format": "uv run ruff format && uv run ruff check --fix",
+    "backend:check:types": "uv run pyright .",
```

Comment:
Somehow `check-types` does not convey the same meaning as `typecheck`, in the language compilation sense.
Proposal: call this `backend:check:type-safety` instead.